chat-plugins/info.js

```diff
@@ -1498,7 +1499,7 @@ var commands = exports.commands = {
 		if (!this.canBroadcast()) return;
 		this.sendReplyBox(
 			"An introduction to the Create-A-Pok&eacute;mon project:<br />" +
-			"- <a href=\"https://www.smogon.com/cap/\">CAP project website and description</a><br />" +
+			"- <a href=\"https://www.smogon.com/cap\">CAP project website and description</a><br />" +
 			"- <a href=\"https://www.smogon.com/forums/showthread.php?t=48782\">What Pok&eacute;mon have been made?</a><br />" +
```

Comment:
The link should change from `https://www.smogon.com/forums/showthread.php?t=48782` to `http://www.smogon.com/forums/threads/48782/`. It still works, even if you don't change it, but it's just to follow the pattern that I've seen.